packages/angular/src/generators/utils/version-utils.ts

```diff
@@ -46,14 +46,48 @@ export function getInstalledAngularVersionInfo(tree: Tree) {
   };
 }
 
+export function getInstalledPackageVersion(
```

Comment:
Should we instead read this information from either the installed package in the node_modules or from the lock files?

I’m concerned about scenarios where someone has “^14.0.0”, but their actual installed version is “14.1.0”.

if we installed “^14.0.0” it would install “14.2.x”.

it may not be a big deal, but just something that I have thought of